docs/worker-node/install-singularity.md

```diff
@@ -28,7 +28,8 @@ releases so it does not require upgrading to 7.6.
 
 The OSG has installed singularity in [OASIS](/worker-node/install-cvmfs),
 so many sites will eventually not need to install singularity locally if
-they enable it to run unprivileged.
+they enable it to run unprivileged.  Meanwhile an RPM installation can
```

Comment:
What do you mean by "eventually"? Does that mean they need to install it locally for unprivileged mode? Or privileged mode? Or both?

Reply:
I can't directly make a comment to lines 12 to 17, but I'd rephrase them as follows:

Kernels with the version 3.10.0-957 or newer include the features necessary to run Singularity as an unprivileged user. This version is the default for RHEL/CentOS/Scientific Linux 7.6 and is available as a security update for previous 7.x releases. This kernel version is not available for RHEL 6 and derivatives.

Without this kernel version, Singularity must be installed with setuid-root executables. Singularity keeps the privileged code to a [minimum](https://www.sylabs.io/guides/2.6/user-guide/introduction.html#security-and-privilege-escalation) in order to reduce the potential for vulnerabilities.

Reply:
By "eventually" I mean "once VOs support it".   I am adding that in a parenthetical comment.

Your suggestion to use "run Singularity as an unprivileged user" isn't quite accurate, since unprivileged users always run it, but I will change it to "run Singularity completely unprivileged".